src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+    private int index;
```

Comment:
Here we do not need to have `index` on class level.
This variable related only to the logic in `getRandomColor()` method.
Its common practice in Java to use such fields only in the method and not create them at class level.

Reply:
OK